DOC-1823: [Partner Ops] Troubleshooting incoming outgoing transfers (#684)
swan-io/docs: docs/topics/operations/guide-troubleshoot-transfers.mdx

:::

---

## Who does what

| Who | What they do |
| --- | --- |
| Your user | Sends and receives transfers, gives consent, contacts the beneficiary or sender, and provides supporting documents. |
| You (Partner Ops) | Identify and classify the transfer in the Dashboard, explain statuses to your user, gather evidence, and contact Swan Support when needed. |
| Swan Support | Processes transfers, reviews flagged transactions, and investigates with banking partners when you provide evidence. |

---

## Transfer types

Credit transfers are push payments: the debtor sends funds to a creditor. Troubleshooting depends on identifying the right type first.

| Transfer type | What it's for | What to know for support |
| --- | --- | --- |
| Instant SEPA Credit Transfer | EUR transfer that processes immediately. | Completes in around 10 seconds. Runs 24/7, including weekends and SEPA closing days. Investigate any `Pending` status. |
| SEPA Credit Transfer | EUR transfer within SEPA, sometimes called a regular transfer. | Settles within 48 business hours, typically 1 to 2 business days. Not processed on weekends or SEPA TARGET closing days. |

> | Swan Internal Credit Transfer | Transfer between two Swan account.s | Almost as quick as instant. Both accounts are on Swan, so there's no external bank to chase. |

Reviewer: ```suggestion
| Swan Internal Credit Transfer | Transfer between two Swan accounts. | Almost as quick as instant. Both accounts are on Swan, so there's no external bank to chase. |
```

For what each transaction status means, see [Credit transfers > Statuses](/topics/payments/credit-transfers/#statuses).

---

## Permissions

Check an account member's rights in **Dashboard** > **Data** > **Account memberships**, then open the membership and check **General** > rights:

- `canViewAccount`: needed to see transfers.
- `canInitiatePayments`: needed to send transfers.
- `canManageBeneficiaries`: needed to add or edit saved beneficiaries.

Adding a beneficiary also requires your user to give consent, separately from consenting to the transfer itself.

---

## Country coverage and currencies

International Credit Transfers go to a limited set of countries, and each currency has to travel on a specific rail. SWIFT carries USD, EUR, and GBP only. Local rails carry local currencies, and your user needs the exact rail for the currency; Indonesian rupiah can only go on the Indonesian rail, for example.

> Check current coverage in [International Credit Transfers](/topics/payments/credit-transfers/international/#outgoing-allowed-countries). If the country or currency isn't supported, Swan can't send or receive funds for that route.

Reviewer: ```suggestion
Check current coverage in [International Credit Transfers](/topics/payments/credit-transfers/international/#risk-outgoing). If the country or currency isn't supported, Swan can't send or receive funds for that route.
```
The heading "Outgoing allowed countries" has the explicit anchor `{#risk-outgoing}`; `#outgoing-allowed-countries` 404s to the top of the page.

| Outgoing | SEPA | `Booked`, beneficiary says not received | Wait 48 business hours from submission. If it's still missing, get written proof of non-receipt. |
| Outgoing | SEPA or International | `Pending` | If a review is indicated, wait. If a technical pending status persists past the next business day, submit a Swan Support request. |
| Outgoing | Any | No transaction at all | Check consent first. If the payment is `ConsentPending`, your user needs to complete consent or start again. |
| Incoming | Instant SEPA | `Booked` | Confirm the funds are available. |
| Incoming | Instant SEPA | `Pending` | Unusual for an instant transfer. Submit a Swan Support request. |
| Incoming | SEPA | Not visible | Ask your user for proof of transfer. If beneficiary details match and normal timelines have passed, the sender's bank needs to investigate. |
| Incoming | International | Not visible | Ask your user for the MT103. Submit a Swan Support request if you still can't locate it. |

---

## Handle outgoing transfers

### If an outgoing transfer is `Booked` but the beneficiary says they didn't receive it

> **Applies to:** SEPA, Instant SEPA, and International Credit Transfers. For Swan Internal transfers, check the receiving Swan account directly instead, there's no external bank involved.

Reviewer: ```suggestion
**Applies to:** SEPA, Instant SEPA, and International Credit Transfers. For Swan Internal transfers, check the receiving Swan account directly instead. There's no external bank involved.
```

A `Booked` outgoing transfer means Swan processed and submitted it.

1. Confirm the transfer details with your user.
2. Direct your user to ask the beneficiary to check with their financial institution.
3. Give your user the transaction confirmation statement or end-to-end reference to pass on. See [Give your user proof of a transfer](#give-your-user-proof-of-a-transfer).
4. If the beneficiary's bank says it didn't receive the funds, direct your user to ask that bank for written proof.
5. Submit a Swan Support request with that proof and the transaction details.

Allow the normal processing time before escalating: 48 business hours for SEPA, around 10 seconds for Instant SEPA, and up to 5 business days for International.

See [Trace a transfer that wasn't received](#trace-a-transfer-that-wasnt-received) for the full procedure.

### If an outgoing transfer is `Rejected`

First check whether the rejection reason is safe to share.

| Reason type | What to say |
| --- | --- |
| Compliance or regulatory | Confirm the transfer was rejected, but don't share more. Explain that regulatory requirements prevent Swan from disclosing the specific reason. |

> | Everything else | Share the specific reason if it's safe to disclose; insufficient funds, invalid recipient details, or a spending limit exceeded. |

Reviewer: ```suggestion
| Everything else | Share the specific reason if it's safe to disclose: insufficient funds, invalid recipient details, or a spending limit exceeded. |
```

Swan checks the account's `Available` balance immediately before executing, and rejects instantly if it's short. `Available` excludes amounts already committed, pending card authorizations, other upcoming transfers, and pending direct debits, so your user may be looking at a different number.

1. Check the `Available` balance at the time of the rejection, not now.
2. Look for other transactions competing for the same funds around that time.
3. Explain the difference, and tell your user to resubmit once funds are available.

### If an outgoing transfer is `Pending`

| Pending reason | What it means | What you do |
| --- | --- | --- |
| Under review | Swan needs more time to review the transfer. | Tell your user it's still processing. Submit a Swan Support request if there's no update after a few business days. |
| Technical or bank execution | The transfer is in an intermediate processing status. | For SEPA or International, this can be normal briefly. If it's still stuck the next business day, submit a Swan Support request. |
| Instant SEPA | Instant transfers complete in around 10 seconds. | Submit a Swan Support request with the transaction details. |

:::note SEPA closing days

> Regular SEPA Credit Transfers aren't processed at weekends or on SEPA closing days: New Year's Day, Good Friday, Easter Monday, Labor Day, Christmas Day, and Boxing Day. Transfers submitted on those days are processed on the next business day. Instant SEPA Credit Transfers are unaffected and run every day of the year.

Reviewer: ```suggestion
Regular SEPA Credit Transfers aren't processed on weekends or on [TARGET closing days](/topics/payments/#sepa-availability): New Year's Day, Good Friday, Easter Monday, 1 May (Labor Day), Christmas Day, and 26 December. Transfers submitted on those days are processed on the next business day. Instant SEPA Credit Transfers are unaffected and run every day of the year.
```
Aligns with the canonical SEPA availability list ("1 May (Labor Day)", "26 December" — "Boxing Day" is British English), uses the canonical term "TARGET closing days", fixes "at weekends" → "on weekends", and links the source of truth.

:::

### If an outgoing transfer isn't visible

> 1. Check whether your user completed consent. If the payment is `ConsentPending`, no transaction exists yet, your user needs to finish consent or start again. Share [Consent and SMS issues](https://support.swan.io/hc/en-150/articles/18130894864541) if consent is failing repeatedly.

Reviewer: ```suggestion
1. Check whether your user completed consent. If the payment is `ConsentPending`, no transaction exists yet: your user needs to finish consent or start again. Share [Consent and SMS issues](https://support.swan.io/hc/en-150/articles/18130894864541) if consent is failing repeatedly.
```

> 2. Check in **Dashboard** > **Data** > **Account memberships** that the member has **Can initiate transfers**.

Reviewer: ```suggestion
2. Check in **Dashboard** > **Data** > **Account memberships** that the member has `canInitiatePayments`.
```
The API permission is `canInitiatePayments`, and the Permissions section of this article (and the memberships page) uses the backticked API names. If the Dashboard checkbox literally reads "Can initiate transfers", keep your bold UI label instead and ignore this.

3. Ask your user to resubmit and complete every validation step.

If your user still can't submit, or the issue looks like it's coming from your integration, see [If your user reports an error or a feature that isn't working](#if-your-user-reports-an-error-or-a-feature-that-isnt-working).

### If your user reports an error or a feature that isn't working

**Applies to:** All transfer types.

Errors when creating a transfer, adding a beneficiary, or setting up a standing order usually come from your product or your integration, not from a transfer that already exists. Don't troubleshoot these as a missing transfer.

1. Ask what they were doing, what they saw, and when, including the exact error message and a screenshot.
2. Check [status.swan.io](https://status.swan.io) for an overlapping incident.

> 3. Check the member's rights in **Dashboard** > **Data** > **Account memberships**. Adding beneficiaries needs **Can manage beneficiaries**; sending needs **Can initiate transfers**.

Reviewer: ```suggestion
3. Check the member's rights in **Dashboard** > **Data** > **Account memberships**. Adding beneficiaries needs `canManageBeneficiaries`; sending needs `canInitiatePayments`.
```
Same as R223: align with the API names used at the top of the article, unless the Dashboard UI labels really read this way.

### If the incoming amount is lower than expected

**Applies to:** International Credit Transfers.

The credited amount can be lower than what the sender says they sent, for two reasons: Swan's processing and currency exchange fees, and intermediary banks deducting their own fees along the routing chain.

Direct your user to [Currency exchange and fees for International Credit Transfers](https://support.swan.io/hc/en-150/articles/20983746712861) for the breakdown. If fees don't explain the difference, submit a Swan Support request.

For a SEPA transfer, fees aren't the explanation. Check for a partial payment or a second transaction instead.

:::note Outgoing international transfers create two transactions

Every outgoing International Credit Transfer produces a transfer transaction and a separate fee transaction. Your user seeing two debits is expected, not a duplicate. You can filter fee transactions in **Dashboard** > **Data** > **Transactions** by the `FeesIn` and `FeesOut` types.

:::

---

## Give your user proof of a transfer

**Applies to:** `Booked` SEPA, Instant SEPA, and International Credit Transfers, incoming and outgoing.

> Users ask for proof of execution, a payment attestation, or a receipt, usually for accounting, or to give to a beneficiary chasing them. Swan provides this as a**transaction confirmation statement**.

Reviewer: ```suggestion
Users ask for proof of execution, a payment attestation, or a receipt, usually for accounting, or to give to a beneficiary chasing them. Swan provides this as a **transaction confirmation statement**.
```

- Download it from **Dashboard** > **Data** > **Transactions**: open the transfer and select **Transaction confirmation**. The button only appears for eligible transfers.

> - It isn't available for transfers in any status other than `Booked`, or for card,direct debit, and fee transactions.

Reviewer: ```suggestion
- It isn't available for transfers in any status other than `Booked`, or for card, direct debit, and fee transactions.
```

:::tip Integration: confirmation statements

Ask your development team to expose transaction confirmation statements in your product using Swan's API, so your users can download their own instead of asking you.

:::

---

## Trace a transfer that wasn't received

**Applies to:** `Booked` SEPA, Instant SEPA, and International Credit Transfers.

Use this when an outgoing transfer is `Booked` but the beneficiary says they haven't received it.

1. Give your user the transaction confirmation statement, including the end-to-end reference.
2. Direct your user to ask the beneficiary to check with their financial institution using that reference.
3. If the beneficiary's bank confirms non-receipt, direct your user to ask that bank for written proof.
4. Submit a Swan Support request with that proof and the transaction details.
5. Swan uses the evidence to investigate with its banking partners.

:::caution An account statement isn't enough

An account statement extract doesn't support a trace. Swan needs a document from the receiving institution confirming the money wasn't received.

:::

---

## Returns and recalls

> These are different things, and it's important to understand the differences.

Reviewer: ```suggestion
Returns and recalls are different things, and mixing them up sets the wrong expectations with your user.
```
Nitpick: the original says the same thing twice.

| | What it is | Who starts it | Timing |
| --- | --- | --- | --- |

> | **Return** | Automatic reversal by a bank that can't process the transfer, closed account, wrong BIC, technical error | The receiving bank, automatically | Funds back within 3 business days |

Reviewer: ```suggestion
| **Return** | Automatic reversal by a bank that can't process the transfer: closed account, wrong BIC, or technical error | The receiving bank, automatically | Funds back within 3 business days |
```

| **Recall** | A request to reverse a completed transfer, usually sent to the wrong IBAN | The debtor, via their bank | Beneficiary responds within 15 business days. Not guaranteed |

Recalls work the same way for SEPA Credit Transfers and Instant SEPA Credit Transfers.

### Request a recall for an outgoing transfer

1. Ask your user for the transaction ID, the reason, and supporting evidence.

> 2. Submit a Swan Support request with the recall details, as early as possible, recalls have a submission deadline.

Reviewer: ```suggestion
2. Submit a Swan Support request with the recall details as early as possible. Recalls have a submission deadline.
```
